app/keepers/keepers.go

```diff
@@ -257,6 +262,26 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		AddRoute(ibctransfertypes.ModuleName, appKeepers.TransferStack)
 	// Note: the sealing is done after creating wasmd and wiring that up
 
+	// ICQ Keeper
+	icqKeeper := icqkeeper.NewKeeper(
+		appCodec,
+		appKeepers.keys[icqtypes.StoreKey],
+		appKeepers.GetSubspace(icqtypes.ModuleName),
+		appKeepers.IBCKeeper.ChannelKeeper, // may be replaced with middleware
+		appKeepers.IBCKeeper.ChannelKeeper,
```

Comment:
Can we add a comment for what these args are supposed to be? (Confused why theres two)

Reply:
Passing it twice is common on IBC modules where you want one with the default implementation and one that is being used to actually route packages through (if there's a middleware, you want to route packets through the middleware but may still need access to the default IBC implementation).

In this case, I don't think the second one is being used. So not sure why it's needed. And the first one is only being used to return the correct app version when implementing `GetAppVersion` (which is required by the interface)

Reply:
Oh I see, I imagine this API / standard could be improved, but for another day. SGTM!